### PR TITLE
Fixed README

### PR DESCRIPTION
**Pull request by alexthomas93**
# Description

Fixes KG creation pipeline example in README.

## Type of Change
- [ ] New feature
- [X] Bug fix
- [ ] Breaking change
- [ ] Documentation update
- [ ] Project configuration change

## Complexity

Complexity: Low

## How Has This Been Tested?
- [ ] Unit tests
- [ ] E2E tests
- [X] Manual tests

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [X] Documentation has been updated
- [ ] Unit tests have been updated
- [ ] E2E tests have been updated
- [ ] Examples have been updated
- [ ] New files have copyright header
- [X] CLA (https://neo4j.com/developer/cla/) has been signed
- [X] CHANGELOG.md updated if appropriate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,13 +84,15 @@ kg_builder = SimpleKGPipeline(
     embedder=embedder,
     entities=entities,
     relations=relations,
-    on_error="CONTINUE",
+    on_error="IGNORE",
```
Comment (stellasia) [inline README.md:87]: We could also remove it since it is the default behavior now, up to you.

Comment (alexthomas93) [reply README.md:87]: Should the default not be to error out?

Comment (stellasia) [reply README.md:87]: We thought the user experience would be nicer if they do not have to deal with LLM-related errors on the first run.

**Review by willtai — APPROVED** [review]
LGTM! Will leave it up to you whether to leave `on_error` in the README example or not

File: README.md (hunk 1 above)
Comment (stellasia) [reply README.md:87]: Not a big deal, can stay as it is.

**Pull request merged**